arch-template-parallels

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Good call, I had removed the extension when pushing to packer because it was building Parallels, but afterwards noticed that I hadn't actually removed it from the base template which meant the extension wasn't the problem.